app/serializers/user_serializer.rb

```diff
@@ -31,7 +31,7 @@ def services
       BackendServices::EDUCATION_BENEFITS
     ]
     service_list += [BackendServices::RX, BackendServices::MESSAGING] if object.can_access_mhv?
-    service_list << BackendServices::DISABILITY_BENEFITS if object.can_access_evss?
+    service_list << BackendServices::EVSS_BENEFITS if object.can_access_evss?
```

Comment:
That would require 4 updates vs 3

1. Update here to use different constant name
2. Update front end to use either value
3. Update backend to send new value
4. Update front end to only use new value

vs.

1. Update here to send 2 values
2. Update frontend to use new value
3. Update backend to only send new value